Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM ubuntu:18.04
+FROM buildpack-deps:jammy
```

Comment:
Couldn't we use `ubuntu:22.04` here and add a few packages to the `apt-get install` command? `buildpack-deps` is a significantly slower download; it also adds much more development packages than we need.

Reply:
The MySQL-python need compilation, so I need to use it.
![image](https://user-images.githubusercontent.com/249085/224494202-f400db95-a3f5-4109-81f0-68a2d01fe7a8.png)

Reply:
Yes, I updated my comment - you just need to add `gcc`.

Reply:
Updated tu Ubuntu 22.04.
What we gain in one place, we lose it the other.